src/text/index.tsx

```diff
@@ -47,19 +48,25 @@ const Text = ({
   children,
   display = TextDisplayType.BODY,
   tag = TextTagType.SPAN,
+  textColor,
   newlineToBr = true,
 }: TextProps) => {
   const baseClassName = 'kirk-text'
   const displayClassName = `${baseClassName}-${display}`
   const Tag = tag
 
+  const cssColorRegex = /^#([0-9a-f]{3}|[0-9a-f]{6})$/i
+
   let content = children
   if (typeof children === 'string' && newlineToBr) {
     content = replaceNewLineWithBR(children)
   }
 
   return (
-    <Tag className={cc([baseClassName, displayClassName, className])}>
+    <Tag
+      className={cc([baseClassName, displayClassName, className])}
+      style={cssColorRegex.test(textColor) ? { color: textColor } : null}
```

Comment:
You could print a warning if the value doesn't match

Reply:
You mean a `console.warn` or `console.error`?

Reply:
hey @pith any further opinion on this?